Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser_test.py

```diff
@@ -119,7 +119,8 @@ def test_suppress_errors(mocker):
     from MicrosoftGraphUser import unblock_user_command, disable_user_account_command, \
         update_user_command, change_password_user_command, delete_user_command, \
         get_direct_reports_command, get_manager_command, assign_manager_command, \
-        revoke_user_session_command, MsGraphClient
+        revoke_user_session_command, get_tap_policy_list_command, \
+        delete_tap_policy_command, create_tap_policy_command, MsGraphClient
```

Comment:
let's add a unit test for each new command to verify that the outputs and the HR are as expected.

Reply:
@Shir2611 - you probably missed that one as well.